spec/facades/food_data_facade_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe FoodDataFacade do
```

Comment:
Good job testing here. I like that you are testing that the Facade is returning an array and the class of each item in the array and nothing more. The facade doesn't care about more than that and it can be easy to overtest this stuff.